src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParser.NetCoreApp.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Data.SqlClient
+{
+    internal sealed partial class TdsParser
+    {
+        internal static void GetGuidBytes(Guid guid, Span<byte> buffer)
```

Comment:
Nit: All these functions could be written with lambda notation 
```
internal static void GetGuidBytes(Guid guid, Span<byte> buffer) => guid.TryWriteBytes(buffer);
```

Reply:
Done where applicable.